.github/ISSUE_TEMPLATE/2-enhancement.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Enhancement / Feature Request
+about: Improving upon existing behavior.
+labels: enhancement
+---
+
+:exclamation: **Please help us keep the number of duplicated issues small!** Have a look at the existing [feature requests](https://github.com/symflower/eval-dev-quality/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement), our latest [roadmap](https://github.com/symflower/eval-dev-quality/issues?q=is%3Aopen+is%3Aissue+label%3Aroadmap) and [open pull requests](https://github.com/symflower/eval-dev-quality/pulls). We kindly ask you to add your suggestions/concerns there in case the feature is already being tracked.
+
+## Problem
+
+> Please describe the problem/inconvenience to be solved.
```

Comment:
Please no "problem/inconvenience " ... write a sentence this/is/not/that/readable and choices are not good. Keep it simple, simple wording.